plugins/org.komodo.eclipse.workspace/src/org/komodo/eclipse/workspace/RelationalConstants.java

```diff
@@ -0,0 +1,159 @@
+/*
+ * JBoss, Home of Professional Open Source.
+ *
+ * See the LEGAL.txt file distributed with this work for information regarding copyright ownership and licensing.
+ *
+ * See the AUTHORS.txt file distributed with this work for a full listing of individual contributors.
+ */
+package org.komodo.eclipse.workspace;
+
+/**
+ * 
+ *
+ * @since 8.0
+ */
+@SuppressWarnings("javadoc")
+public interface RelationalConstants {
+    final static String XMI_EXT = ".xmi"; //$NON-NLS-1$
```

Comment:
Should this be necessary, it could extend StringConstants and thereby reduce some duplications.